api/v1/Game.php

```diff
@@ -805,15 +803,11 @@ public function GetGameDetails(): array|PromiseInterface
             }
 
             $realtimePerEra = explode(",", $state["game_planning_era_realtime"]);
-            // todo: division by zero.
-            $currentEra = intval(floor($state["game_currentmonth"] / $state["game_planning_gametime"]));
-            $realtimePerEra[$currentEra] = $state["game_planning_realtime"];
-            $secondsPerMonthCurrentEra = round($state["game_planning_realtime"] / $state["game_eratime"]);
-            $monthsRemainingCurrentEra = $state["game_eratime"] - $state["game_planning_monthsdone"];
-            $totalRemainingTime = $monthsRemainingCurrentEra * $secondsPerMonthCurrentEra;
+            $totalRemainingTime = intval($state["game_planning_realtime"]);
+            $currentEra = intval($state["game_currentmonth"] / $state["game_eratime"]);
             $nextEra = $currentEra + 1;
             while (isset($realtimePerEra[$nextEra])) {
-                $totalRemainingTime += $realtimePerEra[$nextEra];
+                $totalRemainingTime += intval($realtimePerEra[$nextEra]);
```

Comment:
Is this intval necessary ?
I see "7200,7200,7200,7200" as value for db field value game_planning_era_realtime

So exploding this , would just give an array $realtimePerEra of ints already right  ? And I do not see any adaption of the $realtimePerEra array to make them non ints.